design-documents/graph-ql/coverage/b2b/company.md

```diff
@@ -120,6 +120,7 @@ union CompanyStructureEntity = CompanyTeam | Customer
 type CompanyStructureItem @doc(description: "Company Team and Customer structure") {
     uid: ID! @doc(description: "ID of the item in the hierarchy")
     parent_id: ID @doc(description: "ID of the parent item in the hierarchy")
+    structure_id: ID! @doc(description: "ID of the company structure")
```

Comment:
can you replace structure id and parent id with parent_uid and company_structure_uid? 

Also Ideally instead of parent_uid we would have the originating object name there somewhere  contained in this 'parent_uid' field
this schema needs to represent the future version